DDDEastAnglia/Controllers/SponsorModelQuery.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.DataAccess;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+using DDDEastAnglia.Models;
+
+namespace DDDEastAnglia.Controllers
+{
+    public class SponsorModelQuery
+    {
+        private readonly ISponsorRepository sponsorRepository;
+
+        public SponsorModelQuery(ISponsorRepository sponsorRepository)
```

Comment:
So we have a query object wrapping a repository - that feels like a layer of abstraction too many. I'd like us to move to query objects (and introduce query objects for new work we do), but I don't see any value in using them to wrap a repository. My understanding of them is that they're more like SRP-compliant repositories: they talk directly to the ORM/data access client, but only do One Thing™ rather than All The Data Accesses.